src/code42cli/file_readers.py

```diff
@@ -28,6 +28,12 @@ def read_csv(file, headers):
     else error is raised.
     """
     lines = file.readlines()
+
+    # check if header is commented for flat-file backwards compatability
+    if lines[0].startswith("#"):
```

Comment:
This seems like it could be a little bit hacky, might just be better to either strip the comment line (this would require them to have no unnecessary columns) or throw a message.

Reply:
Since the flat file expectation was only a single column anyway, it would have failed if they had extra columns before this change. So maybe just removing the commented line would be best.